test/function/config/wel_supported.cpp

```diff
@@ -335,6 +335,47 @@ namespace {
     EXPECT_ERROR_ID_EQ( STUMPLESS_ARGUMENT_EMPTY );
   }
 
+  TEST_F( WelSupportedTest, InvalidUtf8InsertionString ) {
+    const struct stumpless_entry *entry_result;
+    LPCSTR invalid_utf8_string = "\xc3\x28 invalid";
+
+    entry_result = stumpless_set_wel_insertion_string( insertion_entry, 0, invalid_utf8_string );
```

Comment:
This line and a few others are over 80 characters long: reformat where you can to bring these to 80 characters or under. There are examples elsewhere in this file of how to format long function calls across multiple lines.